### PR TITLE
Turn exercise 028 to practice

### PR DESCRIPTION
**Pull request by Ozyugoo**

### CODE REVIEW

**Comment by Ozyugoo** [comment]
Hi @cuihtlauac, kindly look at this.

File: practice/028/ex.ml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+module type Testable = sig
+  val length_sort : 'a list list -> 'a list list
+end
+
+module Make(Tested: Testable) : sig val v : test end = struct
+  let v = "length_sort" >::: [
+    "nil" >:: (fun _ -> assert_equal [[]] (Tested.length_sort [[]]));
+    "cons" >:: (fun _ -> assert_equal [["o"]; ["d"; "e"]; ["d"; "e"]; ["m"; "n"]; ["a"; "b"; "c"]; ["f"; "g"; "h"];
+    ["i"; "j"; "k"; "l"]] (Tested.length_sort [["a"; "b"; "c"]; ["d"; "e"]; ["f"; "g"; "h"]; ["d"; "e"];
+    ["i"; "j"; "k"; "l"]; ["m"; "n"]; ["o"]]));
```
Comment (cuihtlauac) [inline practice/028/ex.ml]: Find better names

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: practice/028/ex.ml (hunk 1 above)
Comment (Ozyugoo) [reply practice/028/ex.ml]: Please do check if these names are better names.

**Review by cuihtlauac — APPROVED** [review]
Thnaks @Ozyugoo

**Pull request merged**